### ISSUE TITLE
Download has incomplete status if 'content-length' is not present

### ISSUE THREAD

**Issue by Kuzzy** [hgouveia/node-downloader-helper#60]
If for some reasons target server will not return 'content-length' header for downloaded file, download info object will always have `incomplete: true` property, while file will be downloaded and written to filesystem. (https://github.com/hgouveia/node-downloader-helper/blob/master/src/index.js#L514)

How to reproduce:

```javascript
let result
const url = 'https://www.learningcontainer.com/wp-content/uploads/2020/07/Sample-JPEG-Image-File-Download-scaled.jpg'
const dl = new DownloaderHelper(url, __dirname)

dl.on('end', (downloadInfo) => {
  result = downloadInfo
})

await dl.start()

console.log(result)

// Output: 
// {
//  fileName: 'Sample-JPEG-Image-File-Download-scaled.jpg',
//  filePath: '/my/dir/path/Sample-JPEG-Image-File-Download-scaled (2).jpg',
//  totalSize: NaN,
//  incomplete: true,
//  onDiskSize: 221472,
//  downloadedSize: 221472
// }
```

I think, that this behavior is wrong. 'content-length'  header check should be optional because not every server will respond with this header.

**Comment by hgouveia** [comment]
@Kuzzy agree, i will check this , i need to think a way of how to handle this scenarios properly,  on top of my head i can think of set `totalSize` as -1 value, when failed to fetch the `content-length` , and set `incomplete` to false, since this can't be determined

**Comment by Chaphasilor** [comment]
How about setting `totalSize` to `null`? The total size is "undefined" in this case, and `null` is the explicit version of `undefined` :)

**Comment by hgouveia** [comment]
Released on `2.0.1`
